client/src/pages/queries/components/chart-tab.tsx

```diff
@@ -0,0 +1,159 @@
+            <div className="flex items-center justify-center space-x-4">
+              <div className="flex flex-col items-center justify-center p-8 border rounded-lg">
+                <LineChartOutlined className="text-3xl text-gray-400" />
+                <div className="text-xs">Line Chart</div>
+              </div>
+              <div className="flex flex-col items-center justify-center p-8 border rounded-lg">
+                <AreaChartOutlined className="text-3xl text-gray-400" />
+                <div className="text-xs">Area Chart</div>
+              </div>
+              <div className="flex flex-col items-center justify-center p-8 border rounded-lg">
+                <PieChartOutlined className="text-3xl text-gray-400" />
+                <div className="text-xs">Pie Chart</div>
+              </div>
+              <div className="flex flex-col items-center justify-center p-8 border rounded-lg">
+                <BarChartOutlined className="text-3xl text-gray-400" />
+                <div className="text-xs">Bar Chart</div>
+              </div>
+            </div>
+            <div className="text-gray-500 ">Run query to plot charts</div>
+          </div>
+        </>
+      )
+    }
+
+    if (data) {
+      return (
+        <div className="flex w-full h-full">
+          <div className="flex flex-col w-full px-8 pb-8 space-y-4">
+            {label && values ? (
+              <>
+                <div className="text-center">{title}</div>
+                {cloneElement(chartList[chartType].chart, { data: chartData })}
+              </>
+            ) : (
+              <Empty description={<span>Select data to plot chart</span>} />
+            )}
+          </div>
+          <Form layout="vertical" className="w-96 h-full py=4 pl-4 border-l">
+            <Form.Item label="Chart Type">
+              <Select
+                showSearch
+                value={chartType}
+                onChange={(value) => {
+                  setChartType(value)
+                }}
+              >
+                {chartGroups.map((group) => (
+                  <Select.OptGroup label={group} key={group}>
+                    {chartOrder
+                      .filter((chart) => chartList[chart].group === group)
+                      .map((chart) => (
+                        <Select.Option key={chart} value={chart}>
+                          <div className="flex items-center space-x-3">
+                            {chartList[chart].icon}
+                            <span>{chartList[chart].label}</span>
+                          </div>
+                        </Select.Option>
+                      ))}
+                  </Select.OptGroup>
+                ))}
+              </Select>
+            </Form.Item>
+            <Form.Item label="Label">
+              <Select
+                placeholder="Add label"
+                className="w-full"
+                value={label}
+                onChange={(value) => {
```

Comment:
`onChange=setLabel` would work as fine and there won't be fewer anonymous functions